pom.xml

```diff
@@ -126,6 +126,9 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>3.5.0</version>
+                <configuration>
```

Comment:
Please remove, since this does not seem necessary for fixing the docs.

Reply:
I came across a Javadoc warning without having this configuration.